src/Lib/BSP/IO/GameLumpReader.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.IO;
+using Lumper.Lib.BSP.Lumps;
+using Lumper.Lib.BSP.Lumps.BspLumps;
+using Lumper.Lib.BSP.Lumps.GameLumps;
+
+namespace Lumper.Lib.BSP.IO
+{
+    public sealed class GameLumpReader : LumpReader
+    {
+        private readonly GameLump _gameLump;
+        private readonly long _length;
+        public GameLumpReader(GameLump gamelump, BinaryReader reader, long length)
+            : this(gamelump, reader.BaseStream, length)
+        { }
+        public GameLumpReader(GameLump gamelump, Stream input, long length)
+            : base(input)
+        {
+            _gameLump = gamelump;
+            _length = length;
+        }
+
+        public void Load()
+        {
+            Lumps.Clear();
+            _gameLump.Lumps.Clear();
+            ReadHeader();
+            LoadAll();
+        }
+
+        protected override void ReadHeader()
```

Comment:
I know I'm being so annoying about this, but there's not a single newline throughout this entire function, I really think it harms the readability.

Reply:
Wrong, there are 2 and half of them should be removed :)
Yes I will add a few.